cypress/e2e/po/components/labeled-select.po.ts

```diff
@@ -22,7 +22,9 @@ export default class LabeledSelectPo extends ComponentPo {
   }
 
   clickLabel(label: string) {
-    return this.getOptions().contains('li', label).click();
+    const labelRegex = new RegExp(`^${ label } $`, 'g');
```

Comment:
This caused issues for me when running tests locally because it'd match any dropdown option _containing_ the label provided, eg `clickLabel('default')` might select an option labeled `cluster-fleet-default-c-9jtrc-35cc406cb29a `

Reply:
Was it only happening locally? if so, any idea why? I think I may have run into something similar.

Reply:
I've only seen it locally, I'm guessing when the test runs in ci there aren't other namespaces containing `default` available. There are probably other po methods that are using .contains without a regular expression and brittle in the same way though.